pkg/controllers/cluster/kubeconfig_controller.go

```diff
@@ -52,6 +52,9 @@ func (r *KubeconfigReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 
 	var cluster v1alpha1.Cluster
 	if err := r.Get(ctx, req.NamespacedName, &cluster); err != nil {
+		if client.IgnoreNotFound(err) != nil {
+			return ctrl.Result{}, err
+		}
 		l.Info("skip reconcile, cluster is not found")
 		return ctrl.Result{}, nil
 	}
```

Comment:
Isn't this equivalent to:
```go
if err := r.Get(ctx, req.NamespacedName, &cluster); errors.IsNotFound(err) {
	l.Info("skip reconcile, cluster is not found")
	return ctrl.Result{}, nil
}
```

Reply:
I wanted to return errors other than not-found (rbac, connection or smth else) to the manager